experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -50,14 +50,14 @@ function atmos_init(atmos_config)
 
     if hasmoisture(integrator)
         ρ_flux_q_tot = integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot
-        col_integrated_rain = integrator.p.precipitation.col_integrated_rain
-        col_integrated_snow = integrator.p.precipitation.col_integrated_snow
+        surface_rain_flux = integrator.p.precipitation.surface_rain_flux
+        surface_snow_flux = integrator.p.precipitation.surface_snow_flux
         ᶜS_ρq_tot = integrator.p.precipitation.ᶜS_ρq_tot
         ᶜ3d_rain = integrator.p.precipitation.ᶜ3d_rain
         ᶜ3d_snow = integrator.p.precipitation.ᶜ3d_snow
         @. ρ_flux_q_tot = CC.Geometry.Covariant3Vector(FT(0.0))
-        col_integrated_rain .= FT(0)
-        col_integrated_snow .= FT(0)
+        surface_rain_flux .= FT(0)
+        surface_snow_flux .= FT(0)
```

Comment:
Not related to this PR, but I think `FT` is not needed here.

Reply:
Yeah there are several other locations where we have `Float64`, `FT` where they are likely to be unnecessary - we can address them in a separate PR.